requirements.txt

```
@@ -1,2 +1,3 @@
 pysam==0.15.4
-biopython==1.77
\ No newline at end of file
+biopython==1.77
+pytest
```

```suggestion
pytest==6.2.2
```

Since the other two dependencies are pinned, suggest pinning this one as well